airbyte-cdk/python/airbyte_cdk/sources/declarative/manifest_declarative_source.py

```diff
@@ -98,10 +98,6 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
             )
             for stream_config in self._stream_configs(self._source_config)
         ]
-
-        for stream in source_streams:
-            # make sure the log level is always applied to the stream's logger
-            self._apply_log_level_to_stream_logger(self.logger, stream)
```

Comment:
I didn't dig too much into why, but this doesn't work. It presumably stopped working when we moved the logging logic from the HttpStream to the HttpRequester.

Reply:
The reason is that HttpStream would have it's logger be the parent one i.e. `Stream.logger`. However, HttpRequester does not share it's logger with the stream:
* Stream: [DeclarativeStream](https://github.com/airbytehq/airbyte/blob/d2ccfe72a875414784cc921fe151ffda093d1106/airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_stream.py#L18) which inherits `logger` from `Stream
* [HttpRequester own logger](https://github.com/airbytehq/airbyte/blob/d2ccfe72a875414784cc921fe151ffda093d1106/airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/http_requester.py#L187-L189)